.github/workflows/windows-ci.yml

```
@@ -16,11 +16,24 @@ jobs:
 
     strategy:
       matrix:
-        version: [net462,netcoreapp3.1,net6.0]
+        version: [net462,netcoreapp3.1,net6.0,net7.0]
 
     steps:
     - uses: actions/checkout@v3
 
+    - uses: actions/setup-dotnet@v2
+      with:
+        dotnet-version: '3.1.x'
+
+    - uses: actions/setup-dotnet@v2
+      with:
+        dotnet-version: '6.0.x'
```

nit: can simplify this if desired:

```suggestion
    - uses: actions/setup-dotnet@v2
      with:
        dotnet-version: |
            3.1.x
            6.0.x
```